compose-backstack/src/main/java/com/zachklipp/compose/backstack/Backstack.kt

```diff
@@ -23,14 +21,14 @@ enum class TransitionDirection {
 }
 
 /**
- * Renders the top of a stack of screens (as [T]s) and animates between screens when the top
- * value changes. Any state used by a screen will be preserved as long as it remains in the stack
- * (i.e. result of [remember] calls).
+ * Renders the top of a stack of screens (modeled as [BackstackFrame]s with keys of type [K])
+ * and animates between screens when the top value changes. Any state used by a screen
+ * will be preserved as long as it remains in the stack (i.e. result of [remember] calls).
  *
- * The [backstack] must follow some rules:
+ * The [frames] list must follow some rules:
```

Comment:
"Frames list" is a lot less meaningful than "backstack", since the frame list semantically _is_ a backstack, so I would prefer keeping backstack here.
```suggestion
 * The [backstack][frames] must follow some rules:
```